libs/agno/agno/knowledge/chunking/strategy.py

```diff
@@ -29,3 +30,100 @@ def clean_text(self, text: str) -> str:
         cleaned_text = re.sub(r"\v+", "\v", cleaned_text)
 
         return cleaned_text
+
+
+class ChunkingStrategyType(Enum):
```

Comment:
I don't think the enum should also be a factory. They should be 2 different things.